packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -42,6 +41,7 @@ import * as Amplitude from '../../lib/amplitude';
 import AcceptedCards from '../Product/AcceptedCards';
 
 import './index.scss';
+import PaymentLegalBlurb from '../../components/PaymentLegalBlurb';
```

Comment:
Not going to block on this, but can you file a follow-up to add a story for this component? Could be a "good-first-bug" issue.